molecule/driver/dockerdriver.py

```diff
@@ -223,7 +223,7 @@ def _get_provider(self):
     def _build_ansible_compatible_image(self):
         available_images = [
             tag.encode('utf-8')
-            for image in self._docker.images() for tag in image.get('RepoTags')
+            for image in self._docker.images() for tag in image.get('RepoTags') if tag is not None
```

Comment:
not sure. But it seems to achieve the same thing. Personally, I think the additional if clause is more explicit to the reader... But go ahead with changing it if you feel like it ;-)